{
  "model": "openai/gpt-3.5-turbo",
  "api_key": "<YOUR_OPEN_ROUTER_API>",
  "api_base": "https://openrouter.ai/api/v1",
  "system_context": "<BOT_DESCRIPTION_HERE>",
  "error_message": "Err... :/"
}